style.css

```diff
@@ -517,21 +492,534 @@ form {
   }
 }
 
-.button-submit:hover {
+.button-submit-1:hover {
   animation: bounce 1s;
   cursor: pointer;
   box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
 }
 
-.button-submit:focus {
-  background-color: #008552;
+.button-submit-1:focus {
+  background: #008552;
 }
 
-.my-email {
+.my-email-1 {
   text-align: center;
   font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 20px;
   line-height: 32px;
 }
```

Comment:
Great design @TayyabKM 🤝

- [ ] Kindly style your email on the mobile version properly so that it replicates the one provided in the Figma template.
- Your design
![email](https://user-images.githubusercontent.com/63917725/194352146-5deb82e8-c23a-4c6a-a594-aa9cf803add9.PNG)
- Figma design
![Figma email](https://user-images.githubusercontent.com/63917725/194352333-5b850232-9a22-46b6-aaf7-9f77f4d9b7ed.PNG)

Reply:
Hello @TayyabKM 

- [ ] I wanted you to remove the underline and make the color the same as the one in the Figma template.